<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layout</class>
 <widget class="QDialog" name="layout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stile anpassen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>25</number>
         </property>
         <item row="1" column="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Füllung</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Füllfarbe</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="fillColorSpinBox">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="prefix">
               <string>y = </string>
              </property>
              <property name="minimum">
               <double>-9999999.000000000000000</double>
              </property>
              <property name="maximum">
               <double>9999999.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Füllen bis</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="deleteFillButton">
              <property name="text">
               <string>Füllung löschen</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="fillColorButton">
              <property name="text">
               <string>Färben</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Schatten</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Schattenstil</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="shadowStyleComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Linie</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Punkte (P)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Striche (S)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>P S</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>P P S</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Schattenbreite</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="shadowWidthSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Schattenfarbe</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="shadowColorButton">
              <property name="text">
               <string>Färben</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Linie</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Linienfarbe</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="alphaSpinBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="suffix">
               <string>%</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="lineStyleComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>Linie</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Punkte (P)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Striche (S)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>P S</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>P P S</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Linienstil</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="lineWidthSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Opazität</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Linienbreite</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="lineColorButton">
              <property name="text">
               <string>Färben</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Symbol</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Symbolfarbe</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Symbol</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Symbolrandfarbe</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="symbolSizeSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Symbolgröße</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="symbolComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kreis</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>X</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quadrat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dreieck</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Raute</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Plus</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="symbolColorButton">
              <property name="text">
               <string>Färben</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="symbolPenColorButton">
              <property name="text">
               <string>Färben</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Abbrechen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="styleButton">
           <property name="text">
            <string>Anwenden</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="colorLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
